.gitlab-ci.yml

```diff
@@ -368,95 +368,35 @@ init_workspace:
-    - export RUN_INTEGRATION_TESTS=false
-    # basic tools
-    - apt-get update -q
-    - apt-get install -qqy
-      git wget gnupg2 pass autoconf automake build-essential diffstat gawk chrpath
-      libsdl1.2-dev e2tools nfs-client s3cmd psmisc screen libssl-dev python-dev bash
-      libxml2-dev libxslt-dev libffi-dev nodejs libyaml-dev sysbench texinfo pkg-config
-      zlib1g-dev libaio-dev libbluetooth-dev libbrlapi-dev libbz2-dev libglib2.0-dev
-      libfdt-dev libpixman-1-dev zlib1g-dev jq liblzo2-dev device-tree-compiler
-      qemu-system-x86 qemu-system-arm bc kpartx liblzma-dev cpio sudo awscli gdisk
-    # Prepare mender user
-    - useradd -m -u 1010 mender || true
-    - mkdir -p /home/mender/.ssh
-    - echo "mender ALL=(ALL:ALL) NOPASSWD:ALL" >> /etc/sudoers
-    - sed -i -e 's/^\( *Defaults *requiretty *\)$/# \1/' /etc/sudoers
-    - chown -R mender:mender /home/mender
-    - chown -R mender:mender ${WORKSPACE}
-    - usermod -a -G docker mender
-    # docker-compose from upstream
-    - curl -L "https://github.com/docker/compose/releases/download/1.24.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
-    - chmod +x /usr/local/bin/docker-compose
-    # golang from upstream
-    - wget -q https://storage.googleapis.com/golang/go1.12.linux-amd64.tar.gz
-    - gunzip -c go1.12.linux-amd64.tar.gz | (cd /usr/local && tar x)
-    - ln -sf /usr/local/go/bin/go /usr/local/bin/go
-    - ln -sf /usr/local/go/bin/godoc /usr/local/bin/godoc
-    - ln -sf /usr/local/go/bin/gofmt /usr/local/bin/gofmt
-    # Python 2 pip
-    - apt-get install -qqy python-pip
-    - pip install --upgrade pip
-    - pip2 install requests --upgrade
-    - pip2 install pytest --upgrade
-    - pip2 install filelock --upgrade
-    - pip2 install pytest-xdist --upgrade
-    - pip2 install pytest-html --upgrade
-    - pip2 install -I fabric==1.14.0
-    - pip2 install psutil --upgrade
-    - pip2 install boto3 --upgrade
-    - pip2 install pycrypto --upgrade
-    # Python 3 pip
-    - apt-get install -qqy python3-pip
-    - pip3 install --upgrade pyyaml
+
     # Post job status
     - source ${CI_PROJECT_DIR}/build_revisions.env
     - ${CI_PROJECT_DIR}/scripts/github_pull_request_status pending "Gitlab ${CI_JOB_NAME} started" "${CI_JOB_URL}" "${CI_JOB_NAME}/${INTEGRATION_REV}"
-    # Locales
-    - apt-get install locales
-    - locale-gen --purge en_US.UTF-8
-    - export LC_ALL=en_US.UTF-8
-    - export LANG=en_US.UTF-8
-    - export LANGUAGE=en_US.UTF-8
-    # debugfs
-    - cp /sbin/debugfs /usr/bin/ || echo "debugfs not in /sbin/"
-    # sysstat monitoring suite for Debian/Ubuntu
-    # collect cpu, load avg, memory and io usage every 2 secs forever
-    # use 'sar' from sysstat to render the result file manually
-    - apt-get install -qqy sysstat
-    - sed -i 's/false/true/g' /etc/default/sysstat
-    - service sysstat start
-    - sar -P ALL 2 -o /var/log/sysstat/sysstat.log -uqrbS >/dev/null 2>&1 &
+
     # Setup KVM
```

Comment:
This comment can be removed now.

Reply:
aren't the modprobe commands right below the setup?

Reply:
Well, yeah, but there only nesting VMs is enabled. I assumed the existing comment was related to the install of the kvm software. But feel free to let it stay, it was only a suggestion.

Reply:
should now be gone 👌